src/ui/AvatarSet.ts

```diff
@@ -28,6 +27,15 @@ export default class AvatarSet {
       this.reload();
    }
 
+   public update(src) {
```

Comment:
This function should not be needed, because reload is requesting the avatar and updating all elements.

Reply:
This function is called by the messageevent which allready has the imagedata in it ... i wanted to prevent to much network traffic by reloading again

Reply:
It should not produce any additional network traffic, if you store the avatar at that point. For this reason I removed the function.